tests/tests_reload.yml

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure that crypto_policy_reload variable works
+  hosts: all
+  tasks:
+
+    - name: Run role to make sure all dependecies are installed
+      include_role:
+        name: linux-system-roles.crypto_policies
+
+    - name: Get SSHD pid before policy update
+      slurp:
+        src: /var/run/sshd.pid
+      register: sshd_pid_before
```

Comment:
single blank line after each task

Reply:
@richm Would it be possible to write a yamllint/ansible-lint rule or something to check for this to make this consistent in the all code and not depending on manual review? I am not aware about this recommendation from my reading of the best practices though ...

Reply:
> @richm Would it be possible to write a yamllint/ansible-lint rule or something to check for this to make this consistent in the all code and not depending on manual review?

Probably, I don't know.

> I am not aware about this recommendation from my reading of the best practices though ...

It isn't in the best practices, but probably should be, but only if we can create a yamllint/ansible-lint rule for this.  The only reason for fixing this then would be that all of the other roles usually use the convention of a single blank line between tasks.  But since we don't have a published guideline for this, I'm ok with merging this as-is.

Reply:
Fixed.